README.md

```diff
@@ -1,42 +1,28 @@
-3.	Troubleshooting-R.md
-a.	This file contains three broken chunks of R code which must be investigated and solved. Annotations should be added to clearly layout what was wrong and what was edited to resolve the errors of the original R code. 
-
-Transferring R code to Github
-1.	Copy the URL of the Github repository containing all files of interest.
-2.	Create a new project in R, click “version control”, then click “Git”, paste the URL to the Github repository. Once completed, your files on that repository should be accessible in R studio, select the one you wish to work on.
-3.	Once you have finished your R code, save the script.
-4.	In the Environment section, you can select the checked box beside your recently saved file and then click “commit” 
-5.	A pop up will appear, you should add a comment outlining the recent changes to the R code.
-6.	Once completed, you “Push” your file to transfer the updates to Github
-
-
-
-
-
-
-
-
-
 
+  ![](https://brand.ubc.ca/files/2018/09/1FullLogo_ex_768.png)
 
+### PROJECT MEMBERS
+Carley Winter, Heather Gordon, Bre Lambert and Ramin Rezaeianzadeh
 
 
+##### Project Goals  
+This is the _**Collaborative Project -Milestone 1**_ in conjunction with UBC’s course, STAT 545A. This project has students collaborating through git + GitHub workflow, while also analyzing R code. The goal of this project is to grow students’ skills and confidence when working with R and Github. Milestone 1 consists of three files and eight exercises, all which challenge students to efficiently communicate through Github and trouble shoot broken R code. Detailed instructions on each exercise can be found [here](https://stat545.stat.ubc.ca/collaborative-project/milestone1/). 
 
+#### Files in Repository  
+##### “collaborative-group8”, consists of three files:
 
+Files |  Description
+------|--------------
+1.	__*README.md*__ File |  __a.__ A README file should be present in all projects providing instructions on the tasks ensuring one has a full and detailed understanding of the requirements and goals of the project.  __b.__ The README should contain an explanation of all additional files in the repository.   __c.__ If there is code to be analysed, an instructional guide to analyze the data and transfer it to GitHub should be provided.
+2.	__*TEAMWORK.md *__ File | __a.__	Our TEAMWORK file contains a group contract allowing for expectations to be communicated and agreed upon among members. This document includes how tasks will be divided among group members, clear deadlines for each task, and an agreed upon communication platform that will be used to discuss the project among all group members (e.g. Slack or GitHub Issues).
```

Comment:
_*TEAMWORK.md*_ has an extra space that needs to be removed

Reply:
Extra space is removed. Fixed version pushed to branch. Please review and merge if you approve. @Brelamb